guide/popular-topics/errors.md

```diff
@@ -120,6 +120,24 @@ In this particular example, you can see you are trying to access a piece of data
 
 ## Common discord.js and API errors
 
+### MaxListenersExceededWarning: Possible EventEmitter memory leak detected...
+
+This error is caused by spawning a large number of event listeners, usually for the client. The most common cause of this is nesting your event listeners instead of separating them. The way to fix this error is to make sure you do not nest your listeners; it is **not** to use `emitter.setMaxListeners()` as the error suggests.
+
+You can debug these messages in different ways:
+- Through the [CLI](https://nodejs.org/api/cli.html#cli_trace_warnings): `node --trace-warnings index.js`
+- Through the [`process#warning` event](https://nodejs.org/api/process.html#process_event_warning): `process.on('warning', console.warn);`
+
+### Cannot send messages to this user.
+
+This error throws when the bot attempts to send a DM message to a user but cannot do so. A variety of reasons causes this:
+- The bot and the user do not share a guild (often, people attempt to dm the user after kicking or banning them).
+- The bot tries to DM another bot.
+- The user has blocked the bot.
+- The user has disabled dms in the privacy settings.
+
+In the case of the last two reasons, the error is not preventable, as the Discord API does not provide a way to check if you can send a user a dm until you attempt to send one. The best way to handle this error is to add a `.catch()` where you try to dm the user and either ignore the rejected Promise or do what you want because of it.
```

Comment:
`dm` -> `DM` to enforce consistency.
```suggestion
- The bot and the user do not share a guild (often, people attempt to DM the user after kicking or banning them).
- The bot tries to DM another bot.
- The user has blocked the bot.
- The user has disabled DMs in the privacy settings.

In the case of the last two reasons, the error is not preventable, as the Discord API does not provide a way to check if you can send a user a DM until you attempt to send one. The best way to handle this error is to add a `.catch()` where you try to DM the user and either ignore the rejected Promise or do what you want because of it.
```